src/NServiceBus.Core/Serialization/SerializationFeature.cs

```diff
@@ -8,17 +9,15 @@
     using MessageInterfaces.MessageMapper.Reflection;
     using Pipeline;
     using Serialization;
+    using Settings;
     using Unicast.Messages;
 
     class SerializationFeature : Feature
     {
         public SerializationFeature()
         {
             EnableByDefault();
-            Defaults(s =>
-            {
-                s.SetDefault("AdditionalDeserializers", new List<SerializationDefinition>());
-            });
+            Defaults(s => { s.ApplySerializationDefaults(); });
```

Comment:
probably because you have two overloads on the Getter-method where one does not use the get-or-create approach. Since I suggested to remove that anyway, I think this would also solve the issue here?